include/trompeloeil.hpp

```diff
@@ -3002,6 +3023,19 @@ template <typename T>
     std::abort(); // must never get here.
   }
 
+  template <typename Sig>
+  void
+  report_match(
+    call_matcher_list <Sig> &matcher_list)
+  {
+    std::ostringstream os;
+    for (auto& m : matcher_list)
+    {
+        os << m.name;
+    }
+    send_ok_report<specialized>(os.str());
```

Comment:
I'm not sure I understand the meaning of this string.

Reply:
Leftover when I was debugging compilation error. Will remove it.